source/PlanetPanel.cpp

```diff
@@ -70,6 +70,11 @@ PlanetPanel::PlanetPanel(PlayerInfo &player, function<void()> callback)
 
 void PlanetPanel::Step()
 {
+	if(player.TeleportationStatus() == PlayerInfo::TELEPORTING)
+	{
+		player.SetTeleportStatus(PlayerInfo::MOVED);
+		TakeOff();
```

Comment:
It only teleports in space, not on a planet itself?

Reply:
This Part is a bit complex. I have to have a TakeOff here to remove the old PlanetPanel. But the player will see no TakeOff because the ship gets forced to land in the next frame

Reply:
If its complex it deserves a comment I think

Reply:
Ok

Reply:
done